shipctl/x86_64/Ubuntu_16.04/install.sh

```diff
@@ -19,6 +19,9 @@ cp $SRC_DIR/shippable_replace /usr/local/bin/shippable_replace
 echo "Installing shippable_jdk"
 cp $SRC_DIR/shippable_jdk /usr/local/bin/shippable_jdk
 
+echo "Installing shippable_bump_version"
```

Comment:
Missed this in the earlier review -- `shippable_bump_version` should be part of `utility.sh` as we don't seem to want this to be an independent command.